src/test/java/org/folio/rest/api/ItemStorageTest.java

```diff
@@ -2316,6 +2316,7 @@ private static JsonObject createItemRequest(
     itemToCreate.put("materialTypeId", materialType);
     itemToCreate.put("permanentLoanTypeId", canCirculateLoanTypeID);
     itemToCreate.put("temporaryLocationId", annexLibraryLocationId.toString());
+    itemToCreate.put("_version", 1);
```

Comment:
Why is the client specifying the version? I thought the server generated a version when none was provided by the client.